### PR TITLE
Oculus go integration!

### PR DESCRIPTION
**Pull request by mzguimaraes**

### CODE REVIEW

File: FRL/IO/XRControllerStatus/GVRControllerStatus.cs [hunk 1]
```diff
@@ -31,7 +34,7 @@ public class GVRControllerStatus : XRControllerStatus {
     }
 
     public override bool GetClick(XRButton button) {
-      throw new System.NotImplementedException();
+      return GetPressUp(button);
```
Comment (sebastianherscher) [inline FRL/IO/XRControllerStatus/GVRControllerStatus.cs]: Why is this GetPressUp, not GetPressDown?

File: FRL/IO/XRControllerStatus/GVRControllerStatus.cs [hunk 2]
```diff
@@ -87,6 +90,7 @@ public class GVRControllerStatus : XRControllerStatus {
       if (button == XRButton.Touchpad) return OVRInput.GetUp(OVRInput.Touch.PrimaryTouchpad);
       else return false;
     }
+
```
Comment (sebastianherscher) [inline FRL/IO/XRControllerStatus/GVRControllerStatus.cs]: remove extraneous spaces.

File: FRL/IO/XRControllerStatus/GVRControllerStatus.cs [hunk 3]
```diff
@@ -7,22 +7,25 @@ namespace FRL.IO {
   public class GVRControllerStatus : XRControllerStatus {
 
     private OVRInput.Controller controller;
-
+
```
Comment (sebastianherscher) [inline FRL/IO/XRControllerStatus/GVRControllerStatus.cs]: Remove Tabs and extraneous spaces.

File: FRL/XR/XRManager.cs [hunk 4]
```diff
@@ -50,9 +51,9 @@ public class XRManager : MonoBehaviour {
       if (!(ovr = GetComponent<OVRManager>())) {
         Debug.Log("Adding OVRManager to XRManager gameObject.");
         ovr = this.gameObject.AddComponent<OVRManager>();
-        ovr.trackingOriginType = OVRManager.TrackingOrigin.FloorLevel;
+        ovr.trackingOriginType = OVRManager.TrackingOrigin.FloorLevel;
```
Comment (sebastianherscher) [inline FRL/XR/XRManager.cs]: Remove tabs

File: FRL/IO/XRControllerStatus/GVRControllerStatus.cs (hunk 1 above)
Comment (mzguimaraes) [reply FRL/IO/XRControllerStatus/GVRControllerStatus.cs]: Because when I used GetPressDown trigger input broke for some mysterious reason :/

Comment (sebastianherscher) [reply FRL/IO/XRControllerStatus/GVRControllerStatus.cs]: Explicitly comment that in please :)

**Comment by mzguimaraes** [comment]
Cleaned up everything you asked for boss

**Review by sebastianherscher — APPROVED** [review]

**Pull request merged**